vehicle/renault.go

```diff
@@ -311,17 +318,30 @@ func (v *Renault) kamereonPerson(personID string) (string, error) {
 func (v *Renault) kamereonVehicles(accountID string) ([]string, error) {
 	uri := fmt.Sprintf("%s/commerce/v1/accounts/%s/vehicles", v.kamereon.Target, accountID)
 	res, err := v.kamereonRequest(uri, nil)
-
+	var erroneousVins []string
 	var vehicles []string
 	if err == nil {
 		for _, v := range res.VehicleLinks {
 			if strings.ToUpper(v.Status) == "ACTIVE" {
-				vehicles = append(vehicles, v.VIN)
```

Comment:
Das geht in die falsche Richtung. Entweder geben wir nur gepairte Fahrzeuge zurück, oder wir geben z.B. `Vehicle` Objekte zurück so dass die aufrufende Funktion sich den Pairingstatus anschauen kann.

Ist sicher, dass `ConnectedDriver.Role` das relevante Setting ist?